<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel Principal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos.qrc">
    <normaloff>:/src/vista/imagenes/logoUniConecta.png</normaloff>:/src/vista/imagenes/logoUniConecta.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
		background-color: #cccccc;
    }
    QPushButton {
        background-color: #2e7d32;
        color: white;
        border-radius: 8px;
        font: bold 10pt &quot;Segoe UI&quot;;
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #388e3c;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #388e3c;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutSidebar">
     <item>
      <widget class="QPushButton" name="BotonTablon">
       <property name="styleSheet">
        <string notr="true">      QPushButton:hover {
		background-color: #cccccc;
      }</string>
       </property>
       <property name="text">
        <string>Tablón</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonAnadirEventos">
       <property name="styleSheet">
        <string notr="true">      QPushButton:hover {
		background-color: #cccccc;
      }</string>
       </property>
       <property name="text">
        <string>Añadir Eventos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonGestion">
       <property name="styleSheet">
        <string notr="true">      QPushButton:hover {
		background-color: #cccccc;
      }</string>
       </property>
       <property name="text">
        <string>Gestión</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonCrearTest">
       <property name="styleSheet">
        <string notr="true">      QPushButton:hover {
		background-color: #cccccc;
      }</string>
       </property>
       <property name="text">
        <string>Crear Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonCerrarSesion">
       <property name="styleSheet">
        <string notr="true">      QPushButton:hover {
		background-color: #cccccc;
      }</string>
       </property>
       <property name="text">
        <string>Cerrar sesión</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>600</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century&quot;;
color: #388e3c;</string>
    </property>
    <property name="text">
     <string>Añadir Evento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonAnadirEvento">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Añadir</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="HoraEvento">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="FechaEvento">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2025</year>
      <month>5</month>
      <day>14</day>
     </datetime>
    </property>
   </widget>
   <widget class="QTextEdit" name="UbicacionEvento">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ubicación</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AforoMax">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>361</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Century&quot;;
color: #388e3c;</string>
    </property>
    <property name="text">
     <string>Aforo máximo</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="DescripcionEvento">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Descripción</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="NombreEvento">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
